bundle/language_ar.properties

```diff
@@ -491,6 +491,7 @@ angal.lab.pleaseselectapatient
 angal.lab.positive                                                               = إيجابي
 angal.lab.printreport                                                            = طباعة تقرير
 angal.lab.printtable                                                             = طباعة الجدول
+angal.lab.print																	 = طباعة
```

Comment:
convertion space->tabs happening here

Reply:
Fixed with the push c9b6c019656dba1594baf288c36cf9d381f4130d